reliability/config/enhance_reliability.yaml

```diff
@@ -12,6 +12,13 @@ reliability:
     # for 5 nodes m5.xlarge cluster, 60 is recomended
     maxProjects: 25
     sleepTime: 10
+  cerberusIntegration:
```

Comment:
@qiliRedHat This is a great addition. Cebrerus stores all the failures in a database and we can hit the metrics API to get the history of the failures for the run which in this is the reliability test run duration: https://github.com/cloud-bulldozer/cerberus#metrics-api. It might be useful to save it as an artifact in addition to the logs. Thoughts?

Reply:
Maybe if the status is false we could do a second call specifically to the history api to print out the issues cerberus encountered

Reply:
@chaitanyaenr @paigerube14 Thank you for your comments! I added a function to save the Cerberus history whenever there is new failure items added to the history, if there is no new failure item added, nothing will be done. No mater the Cerberus status check result is `True` or `False` this save function is called always. The reason of doing this instead of saving history when Cerberus status is `False` is, Reliability test checks the Cerberus status in a certain interval, some times the `False` status is recovered to `True` between 2 checks, in this situation, Reliability gets `True` status, but history shows some failure.
The file where the Cerberus history is saved can be configured  or default to /tmp/cerberus-history.json.
PTAL https://github.com/openshift/svt/pull/677/commits/b7ab565fc4e2747ba103b1ce2cca7f7037a268bb

Test I run:
Kill one pod
```
 % oc delete pod node-exporter-nns7m -n openshift-monitoring
pod "node-exporter-nns7m" deleted
```
Cerberus did not hit the `False` status in its check,  but history of the failure is saved
```
2021-07-12 16:22:15,576 - INFO - Cerberus status is 'True'.
2021-07-12 16:22:15,583 - INFO - Cerberus history is updated. Latest history is saved to './cerberus-history.json'
```
```
 cat cerberus-history.json
{
    "history": {
        "failures": [
            {
                "timestamp": "2021-07-12 16:22:11",
                "count": 1,
                "issue": "pod crash",
                "name": "node-exporter-sh8kr",
                "component": "monitoring"
            }
        ]
    }
}%  
```
Kill another pod
```
% oc delete pod node-exporter-5tqm8 -n openshift-monitoring
pod "node-exporter-5tqm8" deleted
```
Cerberus did not hit the `False` status in its check,  but history of the failure is saved
```
2021-07-12 16:23:28,418 - INFO - Cerberus status is 'True'.
2021-07-12 16:23:28,424 - INFO - Cerberus history is updated. Latest history is saved to './cerberus-history.json'

cat cerberus-history.json
{
    "history": {
        "failures": [
            {
                "timestamp": "2021-07-12 16:22:11",
                "count": 1,
                "issue": "pod crash",
                "name": "node-exporter-sh8kr",
                "component": "monitoring"
            },
            {
                "timestamp": "2021-07-12 16:23:24",
                "count": 1,
                "issue": "pod crash",
                "name": "node-exporter-mkptx",
                "component": "monitoring"
            }
        ]
    }
}%   
```